airbyte-integrations/connectors/source-chargify/integration_tests/catalog.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Please fill this file according to the output of running `discover` on the connector.